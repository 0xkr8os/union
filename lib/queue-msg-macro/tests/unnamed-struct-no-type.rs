use queue_msg_macro::queue_msg;

#[queue_msg]
struct Unnamed();

fn main() {}
