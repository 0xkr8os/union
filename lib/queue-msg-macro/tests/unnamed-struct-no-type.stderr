error: only newtype structs are supported
 --> tests/unnamed-struct-no-type.rs:4:15
  |
4 | struct Unnamed();
  |               ^^
